demo/launch/alternative_path_costs.launch.py

```diff
@@ -1,72 +1,28 @@
 import os
```

Comment:
```suggestion

```
Should be able to remove this now that we don't need to `os.path.join` in `load_file` and `load_yaml` anymore